app/controllers/admin_controller.rb

```diff
@@ -8,7 +8,7 @@ def index
 
   def newsletter
     authorize! :manage, :all
-    @members = Member.confirmed.wants_newsletter.all
+    @members = Member.confirmed.undeleted.wants_newsletter.all
```

Comment:
Default scopes tend to make assumptions for one context, at the cost of others and aren't super obvious all of the time. Undeleted... yeah ok, it would make a lot of sense; but it can be tempting to add in default sorts or other bits that are web UI centric. Same sort of scope creep as before hooks can sneak in.

If we do add it, lets prefer a light touch :)